scripts/mod_loader/altered/misc.lua

```diff
@@ -231,3 +235,52 @@ PointList.GetLuaString = function(self)
 	return string.format("CreatePointList(%s)", save_table(extract_table(self)))
 end
 PointList.GetString = PointList.GetLuaString
+
+local spaceDamage = {
+	"bEvacuate",
+	"bHide",
+	"bHidePath",
+	"bSimpleMark",
+	"fDelay",
+	"iAcid",
+	"iDamage",
+	"iFire",
+	"iFrozen",
+	"iPawnTeam",
+	"iPush",
+	"iShield",
+	"iSmoke",
+	"iTerrain",
+	"loc",
+	"sAnimation",
+	"sImageMark",
+	"sItem",
+	"sPawn",
+	"sScript",
+	"sSound"
+}
+
+function SerializeSpaceDamage(self)
+	local result = {}
+	
+	for _, i in ipairs(spaceDamage) do
+		result[i] = self[i]
+	end
+	
+	return result
+end
+
+function DeserializeSpaceDamage(self)
```

Comment:
To follow the naming convention of the other functions:
```suggestion
function CreateSpaceDamage(self)
```